Equilobe.DailyReport.Models/Storage/ReportResendScope.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Equilobe.DailyReport.Models.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Equilobe.DailyReport.Models.Storage
+{
+    public class ReportResendScope 
+    {
+        public long Id { get; set; }
+        public long ReportSettingsId { get; set; }
+        public ResendScope Scope { get; set; }
+        public string UniqueUserKey { get; set; }
+        public DateTime Date { get; set; }
```

Comment:
Pai e nevoie de mai multe date? Ma gandeam ca atunci cand vrei sa executi raportul, se adauga in tabela asta scopul si ceilalti parametrii, si apoi se executa. DateAdded ar fi egal cu DateExecuted in cazul asta